force-app/main/default/lwc/accountModelSettings/accountModelSettings.html

```diff
@@ -7,12 +7,81 @@
                 onsettingssaving={handleSettingsSaving}
                 onsettingssavecompleted={handleSettingsSaveCompleted}
             >
-                <!-- Body of Account Model Settings page (populates Slot in child component) -->
-                <lightning-button
-                    label="Test Button"
-                    disabled={affordancesDisabled}
-                ></lightning-button>
             </c-settings-save-canvas>
+
+            <!-- Body of Account Model Settings page (populates Slot in child component) -->
+            <template if:true={accountModelSettingsViewModel}>
```

Comment:
This is a good solution given how we are populating this variable. Since a lightning spinner for this is not in the spec, that is a valid concern. Depending on page load times we may still need to incorporate one. We can save that as a nice to have.